main.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   # Generic configuration
-  identifier = "cloud-platform-${random_id.name.hex}"
+  identifier = "cp-${random_id.name.hex}"
```

Comment:
Shortening the prefix to reduce the chance of hitting the 64 character limit
I wonder if the service name could replace the random_id part instead?